src/js/plugins/plugin.print.js

```diff
@@ -0,0 +1,79 @@
+/* global BookReader */
```

Comment:
suggestion: add `window`

Reply:
Sorry, Not understand this. for all the recent plugins we used the same I used.